apps/src/gamelab/AnimationPicker/AnimationPreview.jsx

```diff
@@ -56,16 +56,17 @@ const AnimationPreview = React.createClass({
   },
 
   advanceFrame: function () {
-    // Never advance the frame if the frameRate is 0
-    if (this.props.animationProps.frameRate === 0) {
+    // Never advance the frame if the frameDelay is 0
+    if (this.props.animationProps.frameDelay === 0) {
```

Comment:
This seems wrong.  If `frameDelay` is zero, shouldn't we always advance the frame?  Maybe this should be checking against `Infinity`.